test/functional/feature_anchors.py

```diff
@@ -150,98 +134,5 @@ def run_test(self):
-        self.nodes[1].spv_sendrawtx(txinfo['txHex'])
-        self.nodes[1].spv_setlastheight(10)
-        # print ("Anc 1: ", self.nodes[1].spv_listanchors())
-        anc1 = self.nodes[1].spv_listanchors()
-        assert_equal(anc0, anc1)
-
-        print ("Reorg here, 0 & 1 should be equal")
-        self.sync_blocks(self.nodes[0:2], timeout=3)
-        assert_equal(self.nodes[1].getblockcount(), chain0)
-        # time.sleep(2)
-        self.dumpheights()
-        self.dumphashes(block=14)
-
-        print ("Node1: still can generate")
-        self.nodes[1].generate(1)
-        self.sync_blocks(self.nodes[0:2], timeout=3)
-        assert_equal(self.nodes[0].getblockcount(), chain0+1)
-        self.dumpheights()
-        self.dumphashes([0])
-
-        self.dumpheights()
-        self.nodes[2].generate(20)
-        # disconnect_nodes(self.nodes[1], 2)
-        # connect_nodes_bi(self.nodes, 1, 2)
-        time.sleep(1)
-        assert_equal(self.nodes[0].getblockcount(), chain0+1)
-        assert_equal(self.nodes[1].getblockcount(), chain0+1)
-        assert_equal(self.nodes[2].getblockcount(), chain1+20)
-        self.dumpheights()
-        self.dumphashes([2], 14)
-
-        print ("Node2: Setting anchor")
-        self.nodes[2].spv_setlastheight(1)
-        txinfo = self.nodes[2].spv_sendrawtx(txinfo['txHex'])
-        self.nodes[2].spv_setlastheight(10)
-        # print ("Anc 2: ", self.nodes[2].spv_listanchors())
-        assert_equal(anc1, self.nodes[2].spv_listanchors())
-        # disconnect_nodes(self.nodes[1], 2)
-        # connect_nodes_bi(self.nodes, 1, 2)
-
-        self.sync_blocks(self.nodes[0:3], timeout=3)
-        assert_equal(self.nodes[2].getblockcount(), chain0+1)
-
-        # time.sleep(1)
-        self.dumpheights()
-        self.dumphashes([2], 14)
-
-        print ("Node2: Deactivating anchor")
-        self.nodes[2].spv_setlastheight(0)
-        time.sleep(2)
-        # print ("Anc 2: ", self.nodes[2].spv_listanchors())
-        self.dumpheights()
-        self.dumphashes([2], 14)
-        self.dumphashes([2])
-        assert_equal(self.nodes[2].getblockcount(), chain1+20)
-
-        print ("Node2: Reactivating anchor")
-        self.nodes[2].spv_setlastheight(10)
-        # disconnect_nodes(self.nodes[1], 2)
-        # connect_nodes_bi(self.nodes, 1, 2)
-        # print ("Anc 2: ", self.nodes[2].spv_listanchors())
-        self.dumpheights()
-        self.dumphashes([2], 14)
-        self.sync_blocks(self.nodes[0:3], timeout=3)
-        assert_equal(self.nodes[2].getblockcount(), chain0+1)
-        assert_equal(anc1, self.nodes[2].spv_listanchors())
-
-        print ("Node2: Deactivating anchor, no peers")
-        disconnect_nodes(self.nodes[1], 2)
-        self.nodes[2].spv_setlastheight(0)
-        time.sleep(2)
-        # print ("Anc 2: ", self.nodes[2].spv_listanchors())
-        self.dumpheights()
-        self.dumphashes([2], 14)
-        assert_equal(self.nodes[2].getblockcount(), chain1+20)
-
-
```

Comment:
Any downside of removing these tests? Do we have something to replace them?

Reply:
Those test no longer apply, they tested how anchors applied to consensus, in that the client would connect to the anchored chain. That functionality has been removed from the anchoring system.